.github/workflows/ci-commit.yml

```diff
@@ -0,0 +1,32 @@
+name: CI - commit
+
+on:
+  push:
+    branches-ignore:
+      - master
```

Comment:
Seeing this made me realize only 3 modules have "main" branches, all the others have "master". It might be better to set a "master" instead of a "main" for this new module ?

Reply:
Good point. Change is done.